usaspending_api/download/lookups.py

```diff
@@ -77,19 +76,19 @@
     # Transaction Level
```

Comment:
I noticed that "awards" is not being added back to the `VALUE_MAPPINGS` was it originally removed because it was no longer being used even before the download changes?

Reply:
I just remembered the conversation we had about this in the last PR. I think we're good here.

Reply:
Yeh it was no longer being used.